packages/api/test/e2e/nft.e2e.ts

```diff
@@ -433,4 +451,63 @@ describe('NFTs', () => {
 
   });
 
+  it('Get Open listings for Collection', async done => {
+    let reservePrice = 200 * 10_000;
+    let duration = 1000;
+    let token = globalTokenIds[0]
+    let token2 = globalTokenIds[1]
+    // list two out of the three tokens, one auction & one fixed in collection
+    await api.tx.nft.auction(token, spendingAssetId, reservePrice, duration)
+      .signAndSend(tokenOwner, async ({ status }) => {
+        if (status.isInBlock) {
+          await api.tx.nft.sell(token2, null, spendingAssetId, reservePrice, duration)
+            .signAndSend(tokenOwner, async ({ status }) => {
+              if (status.isInBlock) {
+                const allTokens = await api.derive.nft.openCollectionListings(globalCollectionId);
+                // confirm only received 2 out of 3 listings returned and token IDs are correct
+                expect(allTokens.length).toBe(2);
+                const receivedTokenIds = [
+                  {
+                    collectionId: allTokens[0].tokenId.collectionId.toNumber(),
+                    seriesId: allTokens[0].tokenId.seriesId.toNumber(),
+                    serialNumber: allTokens[0].tokenId.serialNumber.toNumber(),
+                  },
+                  {
+                    collectionId: allTokens[1].tokenId.collectionId.toNumber(),
+                    seriesId: allTokens[1].tokenId.seriesId.toNumber(),
+                    serialNumber: allTokens[1].tokenId.serialNumber.toNumber(),
+                  },
+                ]
+                //sort based on series number
+                receivedTokenIds.sort((a, b) => (a.serialNumber > b.serialNumber) ? 1 : -1)
+                expect(receivedTokenIds).toEqual([
+                  {
+                    collectionId: globalCollectionId,
+                    seriesId: 0,
+                    serialNumber: 0,
+                  },
+                  {
+                    collectionId: globalCollectionId,
+                    seriesId: 0,
+                    serialNumber: 1,
+                  }
+                ]);
+                //Ensure Listing Id correctly matches token Id
```

Comment:
```suggestion
                // Ensure Listing Id correctly matches token Id
```
space after `//` (we should have a lint for this)